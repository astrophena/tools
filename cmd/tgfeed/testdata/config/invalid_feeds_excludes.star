feeds = [
    feed(url = "https://example.com/feed.xml"),
    "foobar",
]
