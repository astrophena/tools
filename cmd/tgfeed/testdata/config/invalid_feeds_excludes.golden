[
  {
    "url": "https://example.com/feed.xml"
  }
]